boards/arm/nrf52/arduino-nano-33ble/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# ##############################################################################
```

Comment:
I don't see `arduino-nano-33ble/src/CMakeList.txt`  so there is no point to include this file. Please provide `arduino-nano-33ble/src/CMakeList.txt` or we can remove arduino-nano-33ble/CMakeList.txt from this PR

Reply:
Added.

Reply:
thanks!